packages/core/admin/admin/src/content-manager/components/DynamicTable/CellContent/utils/hasContent.js

```diff
@@ -16,7 +16,12 @@ export default function hasContent(type, content, metadatas, fieldSchema) {
       return content.length > 0;
     }
 
-    const value = content[mainFieldName];
+    const value = content?.[mainFieldName];
+
+    // relations, media ... show the id as fallback
+    if (mainFieldName === 'id' && isNumber(value)) {
```

Comment:
Do we need the `isNumber` value?

Reply:
As discussed on Slack: fixed via 7b4c17dc1410debd0a0f1fe2566055db0e450c08